### PR TITLE
Add blind test to MS

### PR DESCRIPTION
**Pull request by brendonhall**
Test set created by holding one well out of the training data.  MS reworked to accommodate.

### CODE REVIEW

**Comment by kwinkunks** [comment]
I think this looks great, Brendon. Reads great --- I really appreciate the work you have put into this. I feel bad that I sprang the blind thing sort-of last minute, especially considering how on time you were... but it's paid off really nicely. I hope it wasn't too much of a pain to get it done. Anyway, I'll get this off and we will see a proof next week probably.

**Comment by brendonhall** [comment]
Matt - that's great.  I've enjoyed working on this, really appreciate your
feedback and I feel that the article is much stronger because of it.  I'll
need to make some changes to the notebook we'll put in SEG repo, but I
suppose there is a bit of time for that.

I'm looking forward to see how this catches on and the conversations it
starts.  I'm interested in applying these techniques in geoscience and I'll
let you know if I come up with any other ideas.  Feel free to do the same!

I suppose we'll be able to catch up at SEG?

Brendon
